src/services/validator.service.ts

```diff
@@ -0,0 +1,58 @@
+import config from '~app/common/config';
+import { getRequest } from '~root/services/httpApi.service';
+
+const getOwnerAddressCost = async (ownerAddress: string, skipRetry?: boolean): Promise<any> => {
+  try {
+    const url = `${config.links.SSV_API_ENDPOINT}/validators/owned_by/${ownerAddress}/cost`;
+    return await getRequest(url, skipRetry);
+  } catch (e) {
+    return null;
+  }
+};
+
+const clustersByOwnerAddress = async (query: string, skipRetry?: boolean): Promise<any> => {
+  try {
+    const url = `${String(config.links.SSV_API_ENDPOINT)}/clusters/owner/${query}&operatorDetails=operatorDetails&ts=${new Date().getTime()}`;
```

Comment:
I think we should use moment instead of date and always allign with utc (across all our services)

Reply:
moment is deprecated
anyway there is a built in function to get utc timestamp, changed to use it